packaging/packager

```diff
@@ -74,7 +74,7 @@ function package_libc_via_dpkg() {
 function package_libc_via_rpm() {
     if type rpm > /dev/null 2>&1; then
        if [[ $(rpm --query --list glibc | wc -l) -gt 1 ]]; then
-           rpm --query --list glibc | sed -E '/\.so$|\.so\.[0-9]+$/!d'
+           rpm --query --list glibc | sed -E '/\.so$|\.so\.[0-9]+$/!d' | grep lib64
```

Comment:
I'd feel a lot better if we change it to  `grep /lib64/`

Reply:
And add a comment too explaining why.
Thank you!

Reply:
As it turns out there are two packages: `glibc.i686` and `glibc.x86_64`. In the new patch I am using the output of `g++ -dumpmachine` to figure out which one we want to package.

Reply:
1. I don't want people to package 32-bit binaries ever. Lambda runs on x86-64 Linux. Making it run both kinds of binaries is a can of worms that I want to stay away from.

2. $CXX might not be set and clang is the only compiler installed.

I think the solution you had at first is simpler..

Reply:
I also don't think that all distributions that use rpm also use the lib64 convention.